### ISSUE TITLE
[Feature][Accounts For children]Update the sign up form

### ISSUE THREAD

**Issue by anmode** [omegaup/omegaup#6666]
Add a date of birth field in sign up form

**Comment by arishmaurya** [comment]
@anmode  can i take this issue ?

**Comment by anmode** [comment]
> @anmode  can i take this issue ? 

Nope! This issue belongs to my project. But if you wanna suggest the code you can :)
Idk about that u can make PR or not!
